src/solver/simulation/base_post_process.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include "../simulation/sim_structure_probleme_economique.h"
+
+using namespace Antares::Data;
+using namespace Antares::Date;
+
+namespace Antares::Solver::Simulation
+{
+
+
+struct optRuntimeData
+{
+    optRuntimeData(unsigned int y, unsigned int w, unsigned int h) :
+        year(y), week(w), hourInTheYear(h)
+    {}
+
+    unsigned int year = 0;
+    unsigned int week = 0;
+    unsigned int hourInTheYear = 0;
+};
+
+class basePostProcessCommand
```

Comment:
For namings I would suggest to:
 - use consistently `UpperCamelCase` for classes/interfaces
 - remove the words `base` or `interface` from the root of hierarchies: it only hurts readability with longer names without real added value in the context of their use (we don't need to know it's an interface when we use it)
 - In this particular case, I would suggest to remove "command" word from the name, which is most often used for command line: maybe just `PostProcessor` ?